CHANGELOG.rst

```diff
@@ -35,6 +35,8 @@ Fixed
   up correctly, specifically when specifying a bastion host / jump box. (bug fix) #4973
 
   Contributed by Nick Maludy (@nmaludy Encore Technologies)
+* The built-in ``st2.action.file_writen`` trigger has been renamed to ``st2.action.file_written``
```

Comment:
Looks like such a rename would be a better fit the `Changed` category rather then `Fixed`?